.gitignore

```diff
@@ -28,6 +30,9 @@ server/dist/
 ssr/node_modules/
 ssr/dist/
 watcher/node_modules/
+archivecontent/
 content/node_modules/
+content/coverage/
+content/files/
```

Comment:
let's not forget we did this! It's going to git confusing when we finally check some content in :)